README.md

```diff
@@ -216,14 +220,14 @@ onLogout(() => {
 
 ## Vue Router Guards
 
-If you're using Vue Router, you'll often need to guard some routes depending from the user authentication status.
+If you are using Vue Router, you will often need to guard some routes depending on the user authentication status.
 This package provides you some helpers to deal with common scenarios.
 
 ### `authGuard`
 
 Use this helper to create guards relying on the user authentication status.
-The first param your callback receives is a boolean representing the authentication status, while second and third params are the deprecated `to` and `from` Vue Router guards params.
-The returned guard is async as it awaits for the Auth0 client to be initialized before proceeding.
+The first param your callback receives is a boolean representing the authentication status, while the second and third params are the deprecated `to` and `from` Vue Router guards params.
```

Comment:
AFAIK `the` is pretty much always omitted when the subject is plural

Reply:
Well, that's tricky. I cannot find a precise rule or example on this use case. I tried to [look here](https://www.grammarbook.com/grammar/subjectVerbAgree.asp#:~:text=Basic%20Rule.%20A%20singular%20subject%20%28she%2C%20Bill%2C%20car%29,list%20of%20items%20is%20%2Fare%20on%20the%20desk.). But the example is not complete
The point for me is that yes it is a plural subject but composed by two singular subjects.
I also remember to have seen phrases like these in which you are specifying subjects using articles, so the article is required.
When my brother gets back I'll ask his opinion on  this too.
I would also really like ask for example to Kerry or Ross :)

Reply:
My brother says that since the second and third parameter as intended as a block, as a unity of well defined a parameters, then `the` is required

Reply:
Ok, reverted my revertion 👁️ 👅 👁️